models/glif_psc_double_alpha.h

```diff
@@ -0,0 +1,487 @@
+/*
+ *  glif_psc_double_alpha.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef GLIF_PSC_DOUBLE_ALPHA_H
+#define GLIF_PSC_DOUBLE_ALPHA_H
+
+#include "archiving_node.h"
+#include "connection.h"
+#include "event.h"
+#include "nest_types.h"
+#include "ring_buffer.h"
+#include "universal_data_logger.h"
+
+#include "dictdatum.h"
+
+/* BeginUserDocs: integrate-and-fire, current-based
+
+Short description
++++++++++++++++++
+
+Current-based generalized leaky integrate-and-fire (GLIF) models
+(from the Allen Institute)
+
+Description
++++++++++++
+
+``glif_psc_double_alpha`` provides five generalized leaky integrate-and-fire
+(GLIF) models [1]_ with double alpha-function shaped synaptic currents.
+Incoming spike events induce a postsynaptic change of current modeled
+by the sum of two alpha functions (fast and slow components) for each receptor [2]_.
+This function is normalized such that an event of weight 1.0 results in a peak current
+of the fast component of the alpha function to be 1 pA at
+:math:`t = \tau_\text{syn_fast}`.
+The relative peak current of the slow component is given as amp_slow, at
```

Comment:
```suggestion
The relative peak current of the slow component is given as ``amp_slow``, at
```
Make model parameters more distinguishable by highlighting.